src/panel/poi/blocks/Covid19.jsx

```diff
@@ -0,0 +1,77 @@
+import React, { Fragment } from 'react';
+import TimeTable from './TimeTable';
+import covidStrings from './covid_strings';
+import OsmSchedule from 'src/adapters/osm_schedule';
+import OpeningHour from 'src/components/OpeningHour';
+
+// @TODO: refacto OsmSchedule so it doesn't need presentational data
+const scheduleMessages = {
+  open: {
+    msg: 'Ouvert',
+    color: '#60ad51',
+  },
+  closed: {
+    msg: 'Fermé',
+    color: '#8c0212',
+  },
+};
+
+const getContent = (poi, { status, opening_hours, note, contribute_url }) => {
+  const additionalInfo = note &&
+    <div className="covid19-note">
+      <i className="icon-icon_info" />
+      <span dangerouslySetInnerHTML={{ __html: note }}/>
```

Comment:
I see no good reason to inject html here, as `note` contains just text anyway, and is possibly unsafe.